internal/controller/handler.go

```diff
@@ -48,7 +48,7 @@ func (h *Handler) OnAdd(obj interface{}) {
 	// assert the type to an object to pull out relevant data
 	switch obj := obj.(type) {
 	case *corev1.Service:
-		if h.ignored.IsIgnoredService(obj.Name, obj.Namespace) {
+		if h.ignored.IsIgnoredService(obj.Name, obj.Namespace, obj.GetLabels()["app"]) {
```

Comment:
Should we update this to just be:

`if h.ignored.IsIgnoredService(obj.GetObjectMeta())` and have the `IsIgnoredService` parse the meta for name/namespace and labels?

Reply:
Then that way if we change how/what/why meta is parsed, we don't have to modify the handler each time?

Reply:
Yes, I agree with that :)